README.md

```diff
@@ -29,7 +29,7 @@ Ready to dive in? [Explore our taxonomy interactively](https://shopify.github.io
 
 ## 📚 Taxonomy overview
 
-Verticals will be regularly released to this repository, with all 20 published by the end of February, 2024. The following table shows the current status of each vertical.
+The following table shows the current status of each vertical.
```

Comment:
@elsom25 are we okay with this copy change (and above).

Reply:
Now that they're all done, do we just shift to this being a list? Though these copy changes are good to ship and we can come back and do the next pass now that all verticals are in